src/extension.ts

```diff
@@ -498,7 +502,7 @@ function loadKubernetesCore(value : string) {
         }
 
         const filename = value.replace('/', '-');
-        const filepath = path.join(vscode.workspace.rootPath, filename + '.json');
+        const filepath = path.join(vscode.workspace.rootPath || "", filename + '.json');
```

Comment:
rootPath will be deprecated?

Reply:
There are other places using the deprecated workspace API, i prefer to change the legacy code in a new PR.